Libraries/LibC/stdlib.cpp

```diff
@@ -469,6 +470,28 @@ int wctomb(char*, wchar_t)
     ASSERT_NOT_REACHED();
 }
 
+size_t wcstombs(char* dest, const wchar_t* src, size_t max)
+{
+    char* originalDest = dest;
+    while ((size_t)(dest - originalDest) < max) {
+        StringView v { (const char*)src, sizeof(wchar_t) };
+
+        // FIXME: dependent on locale, for now utf-8 is supported.
```

Comment:
I think the plan was for Serenity _not_ to support internationalization, i.e. we can always assume en-US/UTF-8.

(@awesomekling correct me if I'm mistaken)

Reply:
Indeed, localization is a huge timesink and not something that Serenity aims to support.